build-system/compile/bundles.config.js

```diff
@@ -277,3 +283,30 @@ exports.verifyExtensionBundles = function () {
     );
   });
 };
+
+exports.verifyBentoBundles = function () {
+  bentoBundles.forEach((bundle, i) => {
+    const bundleString = JSON.stringify(bundle, null, 2);
+    verifyBundle_(
```

Comment:
nit: for situations in which there are non-obvious positional parameters being passed around, we usually give the param name in comments. Like this:

```
verifyBundle_(
 /* condition */ ...,
 /* field */ ...,
 /* name */ ...,
 /* found */ ... 
)
```

That said, the entire function could be better inlined since imo this is hard to follow and the function is a one-liner. 🤷

Reply:
Yeah... this function took me a bit to understand and I even thought about just rewriting it. but I wasn't sure that change belonged in this PR. I'd like to make this part of a future cleanup.

Reply:
I'm okay with rewriting / simplifying the entire verification logic. It was written at a time when a bug introduced in an extension bundle specification could silently break the build. Separate PR and lower priority, for sure. 👍